index.html

```diff
@@ -7,33 +7,241 @@
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;600;800&display=swap" rel="stylesheet">
   <title>Rhodny's Porfolio Template</title>
 </head>
 <body>
-  <header>
-    <nav class="top-edge-container">
-      <div class="geometry-image"></div>
-      <div class="ic-menu"></div>
-    </nav>
-  </header>
-  <section class="section1-container">
-    <div class="section1">
-      <h1 class="name dark-blue">Charlie Giorgania</h1>
-      <div class="contents">
-        <p class="intro-p dark-blue">
-          Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
-        </p>
-        <p class="intro-p dark-blue">
-          If you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <button class="start-collab bg-dark-blue">Start collaboration</button>
+  <!--Headline section-->
+  <div>
+    <header>
+      <nav class="top-edge-container">
+        <div class="geo-background-img"></div>
+        <div class="ic-menu"></div>
+      </nav>
+    </header>
+    <section class="section-container">
+      <div class="section-sub">
+        <h1 class="title dark-blue">Charlie Giorgania</h1>
+        <div class="contents">
+          <p class="paragraph dark-blue">
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.
+          </p>
+          <p class="paragraph dark-blue">
+            If you like what you see and have a project you need coded, don't hesitate to contact me.
+          </p>
+          <button class="collab-button bg-dark-blue white">Start collaboration</button>
+        </div>
+      </div>
+      <div>
+        <div class="thur-background-img css-background-img"></div>
       </div>
+    </section>
+  </div>
+  
+  <!--Works container 1-->
+  <secion class="section-container">
+    <div class="section-sub">
+      <div><h1 class="title dark-blue">Projects</h1></div>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img.png" alt="This is an image of a person using a laptop." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
```

Comment:
- [ ] Great job on your portfolio so far! However, I noticed that the `buttons` on your page, **including those in the projects and about sections**, do not have the required hover, disable, and active effects. To make your page look more polished and professional, **it's important to add these effects to all of your buttons**. 🙋‍♂️

  ### Figma button effects:
  ![image](https://user-images.githubusercontent.com/107952724/236647408-01484a06-6f5c-45e3-9768-3271a7e72ab0.png)

  -----
- [ ] Additionally, the buttons appear outside of your card, but according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=23-9&t=DOmVpbk2buQv96CX-0), **they should be inside and aligned with the bottom of the card**. Can you please align your buttons to match the design?

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236647458-8c3e90a4-0517-4fbe-92ce-a72c4f8bd114.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236647498-3957d462-f39e-457a-a6b5-7de803b4e703.png)